README.md

```diff
@@ -4,9 +4,9 @@
 
 ## Filter & Sink
 
-Link with a object file who export the symbol `btx_usr_register_callbacks(btx_name_to_dispatcher_t**)`.
+Link with a object file who export the symbol `btx_usr_register_callbacks(void *)`.
 
-The implementation of `btx_usr_register_callbacks` should conssist of calls to `btx_register_callbacks_#{stream_class_name}_#{event_class_name}(name_to_dispatcher, &callbacks)`.
```

Comment:
1. I think at some point you changed from `btx_usr_register_callbacks` to `btx_register_usr_callbacks`. 
2. Same `btx_register_callbacks_#{stream_class_name}_#{event_class_name}(name_to_dispatcher, &callbacks)` to `btx_register_callbacks_#{event_class_name}(name_to_dispatcher, &callbacks)`

Reply:
I think we need to improve the read a lot. I will correct this.

Reply:
Fixed